OpenEdXMobile/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -325,6 +336,7 @@ public boolean tryToSetUIInteraction(boolean enable) {
 
         activityLoginBinding.socialAuth.facebookButton.getRoot().setClickable(enable);
         activityLoginBinding.socialAuth.googleButton.getRoot().setClickable(enable);
+        activityLoginBinding.samlLogin.samlButtonLayout.setClickable(enable);
```

Comment:
I wonder whether the names should be changed here to match the lines above?

`activityLoginBinding.samlAuth.samlButton.getRoot().setClickable(enable);`

Reply:
socialAuth is a layout that  includes two layouts and those layouts include some components but samlLogin includes the components  so  I think it's unnecessary add a layout that includes another layout, in this case  activityLoginBinding.samlLogin.samlButtonLayout.setClickable(enable) is the same  that activityLoginBinding.samlLogin.getRoot.setClickable(enable), this could be a possible option or activityLoginBinding.samlLogin.samlButton.setClickable(enable);